src/vanilla/ClientModelExtensions.cs

```diff
@@ -727,7 +728,8 @@ public static void ConstructMapper(TSObject mapper, IModelType type, string seri
 
             void applyConstraints(TSObject obj)
             {
-                if (constraints != null && constraints.Any())
+                bool useClientSideValidation = (bool) Settings.Instance.CustomSettings["ClientSideValidation"];
```

Comment:
Can you add this to GeneratorSettingsTS? That way we don't have to use the string literal.